README.md

```diff
@@ -1 +1,182 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">Html</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/css/default.asp">css</a></li>
+  </ul>
+</details>
+<details>
+  <summary>js</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/js/">javaScript</a></li>
+  </ul>
+</details>
+<details>
+  <summary>React js</summary>
+  <ul>
+    <li><a href="https://react.dev/">javaScript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Use the high fidality design for the project😊👌
+- Use the semantics elements💯
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A web browser to view output e.g [Google Chrome](https://www.google.com/chrome/).
+- An IDE e.g [Visual studio code](https://code.visualstudio.com/).
+- Install the `npm` package manager use this [to install both node and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
+- [A terminal](https://code.visualstudio.com/docs/terminal/basics).
+
+### Setup
+
+Clone this repository to your desired folder or download the Zip folder:
+
+```
+git clone https://github.com/BilalLiaquat7/metrics.git
+```
+
+- Navigate to the location of the folder in your machine:
+
+**`you@your-Pc-name:~$ cd desktop`**
+
+### Install
+
+To install all dependencies, run:
+
+```
+npm install
+```
+
+### Usage
+
+To run the project, follow these instructions:
+
+- After Cloning this repo to your local machine.
+- To get it running on your default browser and local host, run:
+
+1. Open the index with live server from the code editor
+2. Or you can double click on the index to open it in the browser
+
+### Run tests
+
+To run tests, run the following command:
+
+- Track HTML linter errors run:
+
+```
+npx hint .
+```
+
+- Track CSS linter errors run:
+
+```
+npx stylelint "**/*.{css,scss}"
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🎥 Future Feature <a name="future feature"></a>
+
+- In future i will improve design.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Shah Zaman**
+
+- GitHub: [@githubhandle](https://github.com/BilalLiaquat7/)
+- LinkedIn: [@Linkedin](https://www.linkedin.com/in/muhammad-bilal-liaquat-87863390/)
+
+<p align="right">(<a href="#live-demo">back to top</a>)</p>
+
+## 🎥 Project Presentation
+
+- Project Demo: [Project Demo](https://metrics-8e7g.onrender.com/)
+
+- Project Presentation: [ProjectPresentation](https://www.loom.com/share/bf2e2616af3c4547af0b12303e332e39?sid=53ef421e-a178-407e-beab-dcd0c3371d85)
```

Comment:
- [ ] Great job on making your project presentation under 5 minutes and talking about giving a clear and objective description of the project in professional English, however please kindly add:

- a highlight about a piece of the code that you find interesting or are proud of making.

Remember, you're making good progress!. Keep up the good work!

Reply:
* As mentioned by the previous CR Kindly highlight a piece of the code that you find interesting or are proud of making.

Reply:
Is this mandatory to record a new video?
Couldn't be passed without it? Because I already recorded a video.